File: Examples/VisualBasic/Articles/RenderingAndPrinting/ConvertChartToSvgImage.vb
Namespace Articles.RenderingAndPrinting
    Public Class ConvertChartToSvgImage
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Create workbook object from source file
            Dim workbook As New Workbook(dataDir & Convert.ToString("SampleChartBook.xlsx"))

            ' Access first worksheet
            Dim worksheet As Worksheet = workbook.Worksheets(0)

            ' Access first chart inside the worksheet
            Dim chart As Aspose.Cells.Charts.Chart = worksheet.Charts(0)

            ' Set image or print options
            Dim opts As New Aspose.Cells.Rendering.ImageOrPrintOptions()
            opts.SaveFormat = SaveFormat.SVG

            ' Save the chart to svg format
            chart.ToImage(dataDir & Convert.ToString("Image_out.svg"), opts)
            ' ExEnd:1
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Articles/RenderingAndPrinting/ConvertChartToSvgImage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Cells.Examples.CSharp.Articles.RenderingAndPrinting
{
    public class ConvertChartToSvgImage
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Create workbook object from source file
            Workbook workbook = new Workbook(sourceDir + "sampleConvertChartToSvgImage.xlsx");

            // Access first worksheet
            Worksheet worksheet = workbook.Worksheets[0];

            // Access first chart inside the worksheet
            Aspose.Cells.Charts.Chart chart = worksheet.Charts[0];

            // Set image or print options
            Aspose.Cells.Rendering.ImageOrPrintOptions opts = new Aspose.Cells.Rendering.ImageOrPrintOptions();
            opts.SaveFormat = SaveFormat.Svg;

            // Save the chart to svg format
            chart.ToImage(outputDir + "outputConvertChartToSvgImage.svg", opts);

            Console.WriteLine("ConvertChartToSvgImage executed successfully.");
        }
    }
}
